### PR TITLE
[BOUNTY] Genetic Editing Wills the Smiths

### LINKED ISSUE #632: Code Bounty: Will Smith Mutation
Scrambling DNA at the monkey mutation machine should have a chance of turning of the subject into a Will Smith. Humans share 50% of their DNA with Will Smith, so this should be possible.

Super powers include: increased twitter followers
https://www.bountysource.com/issues/82593794-code-bounty-will-smith-mutation

### PR DESCRIPTION
**Pull request by Kerbin-Fiber**
## About The Pull Request
When scrambling a human's DNA in a DNA computer, there's a 1 in 100 chance that they'll be Willed into a Smith, or, turned into a Will Smith.

See: https://imgur.com/a/iYHv4qN

Closes #632 

## Why It's Good For The Game
Ah yeah, now that's hot. This is a bounty too.

I would like someone to also yell at me since the code is probably spaghetti even though its a few lines.

## Changelog
:cl:
add: Genetic Engineering can now accidentally turn you into Will Smith!
/:cl:

### CODE REVIEW

**Comment by Kerbin-Fiber** [comment]
please tell me if you want the chance of it happening to be lower, or higher. 1 in 100 doesn't seem bad because you don't NEED to use the scramble button that often to find all the genes.

Edit: Unless you're a tot and lock someone in a scanner and kill them with rads and they pop out as Will Smith

File: code/game/machinery/computer/dna_console.dm [hunk 1]
```diff
@@ -554,6 +554,11 @@
 			if(viable_occupant && (scrambleready < world.time))
 				viable_occupant.dna.remove_all_mutations(list(MUT_NORMAL, MUT_EXTRA))
 				viable_occupant.dna.generate_dna_blocks()
+				if(prob(1))
```
Comment (ike709) [inline code/game/machinery/computer/dna_console.dm]:
```suggestion
				if(prob(1) && prob(10))
```
This should be a really low chance IMO.

**Comment by ike709** [comment]
Add a `log_game()` and `message_admins()` message for when this happens.

**Comment by qwertyquerty** [comment]
Do we really want this

**Comment by Kerbin-Fiber** [comment]
Yes, we have gorillas, after all

**Review by MarkSuckerberg — APPROVED** [review]
Don't see why we need it, but a bounty's a bounty `¯\_(ツ)_/¯`

**Comment by qwertyquerty** [comment]
dumb idea, dumb bounty.

**Pull request closed without merging**